onnxruntime/python/tools/transformers/convert_generation.py

```diff
@@ -1374,6 +1390,11 @@ def convert_generation_model(args: argparse.Namespace, generation_type: Generati
     if args.vocab_size != -1:
         vocab_size = args.vocab_size
 
+    if args.eos_token_id != -1:
```

Comment:
nit: May merge the following with
``
    eos_token_id = config.eos_token_id
    pad_token_id = config.eos_token_id if is_gpt2 else config.pad_token_id
``
as
``
eos_token_id = args.eos_token_id if args.eos_token_id >= 0 else config.eos_token_id
pad_token_id = args.pad_token_id if args.pad_token_id >= 0 else (config.eos_token_id if is_gpt2 else config.pad_token_id)
``